bigquery_etl/cli/glean_usage.py

```diff
@@ -31,7 +31,11 @@
     metrics_clients_last_seen.MetricsClientsLastSeen(),
     clients_last_seen_joined.ClientsLastSeenJoined(),
 ]
-SKIP_APPS = ["mlhackweek_search"]
+
+# regrets_reporter currently refers to two applications, skip the glean
+# one to avoid confusion: https://github.com/mozilla/bigquery-etl/issues/2499
+# mlhackweek_search was an experiment that we don't want to generate tables for
```

Comment:
@jklukas do you have any idea why mlhackweek_search is here? It looks like it was added in https://github.com/mozilla/bigquery-etl/pull/2416

Reply:
This is probably due to permissions. `mlhackweek_search` data was considered sensitive and locked down to a specific google group. But I can't exactly remember what the failure scenario was. Likely airflow would have been able to run this, but local testing with an individual dev's account would have failed.

Reply:
Ok, I think the current comment probably gives enough context.